ADALiOS/ADALiOS/ADAuthenticationContext+TokenCaching.m

```diff
@@ -93,7 +94,7 @@ - (ADTokenCacheStoreItem*)findCacheItemWithKey:(ADTokenCacheStoreKey*) key
         else
         {
             //We have a cache item that cannot be used anymore, remove it from the cache:
-            [self.tokenCacheStore removeItemWithKey:key userId:userId error:nil];
+            [self.tokenCacheStore removeItemWithKey:key userId:userId.userId error:nil];
```

Comment:
Unless you feel really strongly on this, I recommend punting to redesign. This would require days of work and would be a breaking change.